db/migrate/20240226104548_change_claims_eligibility_type_to_policies_early_career_payments_eligibility.rb

```diff
@@ -0,0 +1,19 @@
+class ChangeClaimsEligibilityTypeToPoliciesEarlyCareerPaymentsEligibility < ActiveRecord::Migration[7.0]
+  class Claim < ApplicationRecord; end
+
+  def change
+    reversible do |dir|
+      dir.up do
+        Claim.where(eligibility_type: "EarlyCareerPayments::Eligibility").find_each do |claim|
+          claim.update!(eligibility_type: "Policies::EarlyCareerPayments::Eligibility")
+        end
```

Comment:
This will be a lot quicker and will skip validations. In the past we have had issues with validations blocking unrelated updates:

```suggestion
        Claim.where(eligibility_type: "EarlyCareerPayments::Eligibility").update_all(eligibility_type: "Policies::EarlyCareerPayments::Eligibility")
```